package.json

```
@@ -310,7 +310,7 @@
     "proxy-from-env": "1.0.0",
     "proxyquire": "1.8.0",
     "puid": "1.0.7",
-    "puppeteer": "npm:@elastic/puppeteer@5.4.1-patch.1",
+    "puppeteer": "8.0.0",
```

Any reason we don't use a `^` to signal that we can upgrade this within the current major?

```suggestion
    "puppeteer": "^8.0.0",
```